shims/common/src/main/scala/org/apache/gluten/sql/shims/SparkShims.scala

```diff
@@ -207,6 +207,11 @@ trait SparkShims {
 
   def attributesFromStruct(structType: StructType): Seq[Attribute]
 
+  // Spark 3.3 and later only have file size and modification time in PartitionedFile
+  def getFileSizeAndModificationTime(file: PartitionedFile): (Option[Long], Option[Long]) = {
+    (None, None)
+  }
+
```

Comment:
we'd better keep pure abstract method in this file, just like other declaration.

Reply:
Updated, thank you!